packages/sample-app/sample_app/redis_rag_app.py

```diff
@@ -0,0 +1,127 @@
+import os
+import numpy as np
+
+from redis import Redis
+from redis.exceptions import ResponseError
+from redis.commands.search.field import TagField, VectorField, TextField
+from redis.commands.search.indexDefinition import IndexDefinition, IndexType
+from redis.commands.search.query import Query
+from openai import OpenAI
+from traceloop.sdk import Traceloop
+from traceloop.sdk.decorators import workflow
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+
+INDEX_NAME = "index"
+DOC_PREFIX = "doc:"
+EMBEDDINGS_DIM = 1536
+
+Traceloop.init(exporter=ConsoleSpanExporter())
+# Traceloop.init(app_name="redis_rag_app")
```

Comment:
@mpozniak95 maybe keep this line instead of the one above?

Reply:
Done